### ISSUE TITLE
How about the accuracy of triplet loss training?

### ISSUE THREAD

**Issue by tonybaigang** [davidsandberg/facenet#403]
I had try to train the model by myself. 
I download CASIA-WebFace which had been celan and then follow the instructions in wiki "Triplet loss training" to train the mode. 
But it seems the accuracy on LFW, is %88, is it right?

![result](https://user-images.githubusercontent.com/12961153/28765854-67cf6a5a-75ff-11e7-8aa8-f99ecb5d3e6c.png)

**Comment by qixianbiao** [comment]
Obviously, it is very hard to train triplet loss nn.  Until now, except for the original google facenet, I have not seen any case with very sussuccful triplet loss training.  
So I think the performance it is reasonable although I have not trained it.

**Comment by ycdhqzhiai** [comment]
Hi 
I am a neophyte,I don't understand why batch_number exceeds epoch_size.

**Comment by LCrinesnow** [comment]
Yes, I get 89% Accuracy.

**Comment by RakshakTalwar** [comment]
@tonybaigang please look at the wikis, David mentions this

**Comment by lingbao00** [comment]
I am doing the training and i follow the wiki of Triplet loss training,but i do not know how to get the data ~/datasets/casia/casia_maxpy_mtcnnalign_182_160,i aligned the casia data to 160 ,but get error:
Input to reshape is a tensor with 4096 values, but the requested shape requires a multiple of 384.
so i hope someone can help me how to do the training
